client/payment-details/summary/index.tsx

```diff
@@ -42,7 +42,27 @@ const placeholderValues = {
 	refunded: null,
 };
 
-const composePaymentSummaryItems = ( { charge }: { charge: Charge } ) =>
+const isTapToPay = ( model: string ) => {
+	if ( model === 'COTS_DEVICE' ) {
+		return true;
+	}
+
+	return false;
+};
+
+const getTapToPayChannel = ( platform: string ) => {
+	return platform === 'ios'
+		? __( 'Tap to Pay on iPhone', 'woocommerce-payments' )
+		: __( 'Tap to Pay on Android', 'woocommerce-payments' );
```

Comment:
Same as in another PR (I think), it would be great to have separate if/else (or a switch) for those two, and a default one:

```
'ios' => 'Tap to Pay on iPhone'
'android' => 'Tap to Pay on Android'
default => 'Tap to Pay'
```